project/gen2.scala

```diff
@@ -19,6 +17,12 @@ import shapeless.record._
 
 object gen2 {
 
+  final case class Angle(toArcsecs: Double)
+  object Angle {
+    def fromArcsecs(d: Double) =
+      Angle(d)
+  }
```

Comment:
In the build build we don't have access to OCS2's `Angle` anymore, nor gem's `Angle` since it hasn't been built yet. This is just enough structure to read an angle from the database and insert its value into the generated Scala code.